lib/rage/cli.rb

```diff
@@ -25,6 +25,40 @@ def server
 
       ::Iodine.start
     end
+
+    desc 'routes', 'List all routes.'
+    def routes
+      # the result would be something like this:
+      # +----------------------+----------------------+----------------------+
+      # | Action               | Verb                 | Path                 | Controller#Action
+      # | index                | GET                  | /                    | application#index
+
+      # load config/application.rb
+      require File.expand_path('config/application.rb', Dir.pwd)
+      # load config/routes.rb
+      require File.expand_path('config/routes.rb', Dir.pwd)
+
+      routes = Rage.routes.router.routes
+
+      # construct a table
+      table = [['+---------------------------', '+---------------------------', '+---------------------------+']]
```

Comment:
Let's try to mimic the output of `rails routes` but without the first `Prefix` column. That would also mean removing all table formatting.